packages/mdc-icon-button/_mixins.scss

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+@import "@material/theme/mixins";
+@import "./variables";
+
+@mixin mdc-icon-button-size($width, $height: $width, $padding: max($width, $height)/2) {
+  width: $width + $padding * 2;
+  height: $height + $padding * 2;
+  padding: $padding;
+
+  // stylelint-disable-next-line selector-max-type
+  svg,
+  img {
+    width: $width;
+    height: $height;
+  }
+}
+
+@mixin mdc-icon-button-font-size($font-size) {
```

Comment:
I'm still not sure I follow why we need this rather than just adding `font-size` to `mdc-icon-button-size` and making it all one mixin.